vars/sendToSlack.groovy

```diff
@@ -0,0 +1,15 @@
+def call(message, messageType) {
+    
+    def color = '#D3D3D3'
+
+    if (messageType == 'success') {
+        color = '#008000'
+    } else if (messageType == 'inProgress') {
+        color = '#FFFF00'
+    }
+    else if (messageType == 'failure') {
+        color = '#FF0000'
+    }
+    
+    slackSend (color: ${color}, message: ${message}: `${env.JOB_NAME}` [Build #${env.BUILD_NUMBER}] \n(<${env.BUILD_URL}|Open>)")
```

Comment:
do you know that you'll have access to the environment variables in the function?

Reply:
Yes in a def call you have access to them. You dont in a class however.

Reply:
Okay.

Reply:
Test it brother. LGTM